dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -2332,7 +2362,7 @@ public void relateContent(Contentlet contentlet, ContentletRelationshipRecords r
             Set<Tree> uniqueRelationshipSet = new HashSet<Tree>();
 
             Relationship rel = related.getRelationship();
-            List<Contentlet> conRels = getRelatedContent(contentlet,related.getRelationship(), related.isHasParent(), user,respectFrontendRoles) ;
+            List<Contentlet> conRels = getRelatedContentFromIndex(contentlet,related.getRelationship(), related.isHasParent(), user,respectFrontendRoles) ;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 155 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)